### ISSUE TITLE
HTML parser: `Cannot read property 'props' of undefined`

### ISSUE THREAD

**Issue by arraydude** [remarkjs/react-markdown#385]
react-markdown is crashing when trying to render this html:

```html
<div class="entities" style="line-height: 2.5">
<mark class="entity" style="background: #7aecec; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em; box-decoration-break: clone; -webkit-box-decoration-break: clone">

Lorem Ipsum
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; text-transform: uppercase; vertical-align: middle; margin-left: 0.5rem">ORG</span>
</mark>
Testing</br>Testing</br>Testing
</div>
```

**Comment by rwlaschin** [comment]
I'm having a crash as well

Shouldn't this be something like this instead?  
It would make sense to not run this code of el.props.children was null or otherwise invalid
![image](https://user-images.githubusercontent.com/2682324/77857836-d8964480-71b4-11ea-9e6e-d7ee786c7e69.png)

**Comment by ChristianMurphy** [comment]
https://github.com/rexxars/react-markdown/pull/428 would fix this

**Comment by wooorm** [comment]
This may also be fixed on 5.0.0, [with this change](https://github.com/remarkjs/react-markdown/commit/22a5e49ae9dde84257d043f033d1b640b78c7b34#diff-e48a0b02aa7a8f65a453b849e7345933ee0072fb6d0c2a109c83f43cc8fae8c6). Could someone confirm whether that works? Otherwise it’s just the HTML parser being funky and the plan is for that to be replaced next major.

**Comment by rcowsill** [comment]
It still happens at https://remarkjs.github.io/react-markdown/. If I'm reading the history right, that's running 5.0.0 already.

By the way, this looks like a duplicate of #302 and possibly #345 too.

**Comment by cerkiewny** [comment]
https://github.com/remarkjs/react-markdown/pull/507 will fix that

**Comment by wooorm** [comment]
This should be solved by landing GH-563 today, which will soon be released in v6.0.0!
